src/rewards/Forms/hooks/useFormSubmissions.tsx

```diff
@@ -89,7 +89,7 @@ const useFormSubmissions = (formId, queryString) => {
   return { data: flattenedData, ...rest }
 }
 
-const useUserFormSubmission = (form: Schemas["Form"]) => {
+const useUserFormSubmission = (form?: Schemas["Form"]) => {
```

Comment:
nit: is there a reason this is optional now? 👀

Reply:
form can be sometimes undefined (because we get its value from a useSWR hook), that's why it's optional here